park_info.clj

```diff
@@ -1,198 +1,197 @@
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field false
-        :sports_field_extra: nil
-        :playground true
-        :playground_extra: nil
-        ;---Directions---
-        :public_transport "
-        stop: Wenceslaus Sq. - trams: #3, #9, #14, #24, #51, #52, #54, #55, #56, #58 
-        stop: Můstek - metro line(s): A, B"
-        :GPS_coords "50°4′58.8″ N, 14°25′24.8″ E"
-        :parking false
-        :parking_extra nil
-        ;---Additional Information---
-        :dogs false
-        :path_type "paved, special surface in child playpen"
-        :administered_by "Prague City"
-        :opening_hours "Year long, closes at night"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/50/c9/77/184207_4_frantiskanska_mapa.pdf"
-       
-    }
-    def Obora_Hvezda{
-        ;---Basic Info---
-        :food true
-        :food_extra "During Summer Season on the premises of the Villa (www.pamatniknarodnihopisemnictvi.cz), tel:. +420 235 357 938,
-        or at Libocká street, restaurant U Holečků, www.uholecku.cz,
-        or at Libocká studna, tel. +420 777 836 966,
-        alternatively at Petřiny"
-        :toilet true
-        :toilet_extra "Portable toilets on the premises, during summer season toilets are also available in the villa"
-        :POIs "Suitable for tourists and families with children, heritage trees, nature trail, natural attractions"
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field false
-        :sports_field_extra: "nearby: Youth House (www.ddmp6.cz/kontakt/) & traffic playground"
-        :playground true
-        :playground_extra: "2 playgrounds"
-        ;---Directions---
-        :public_transport "
-        stop: Nad Markétou - buses: #179, #184, #191, #510, 
-        stop: Libocká - buses: #179, #510, 
-        stop: Vypich - trams: #15, #22, #25, #57"
-        :GPS_coords "50°5′2.798″ N, 14°19′45.529″ E"
-        :parking true
-        :parking_extra "8 parking spots at Libocká gate, free parking in U Hvězdy street"
-        ;---Additional Information---
-        :dogs false
-        :path_type "hardened soil, threshing floor"
-        :administered_by "Museum of Czech Literature, Prague City"
-        :opening_hours "park - year long, villa - (April 10:00-17:00, May-September 10:00-18:00, October 10:00-17:00, November-March CLOSED) "
-        ;---Map---
-        :map_url "https://www.praha.eu/public/bf/26/3e/184361_4_hvezda_mapa.pdf"
+(def Frantiskanska_Zahrada{
+    ;---Basic Info---
+    :food true
+    :food_extra "In the vicinity - Alfa passage, Wenceslaus & Jungmann square, Narodni Trida"
+    :toilet true
+    :toilet_extra "with wheelchair access"
+    :POIs "Pleasant dining in the historical town district, suitable for both children and the elderly, Church of Our Lady of the Snows"
```

Comment:
Move the 'suitable for...' as the last point, maybe? That way it makes more sense I think: in this park there is X, Y and Z; suitable for A and B.

Reply:
These are literal translations from the website. Don't personally see a reason to change these.